### PR TITLE
feat: support "torrent" in Spotlight indexation

### PR DESCRIPTION
**Pull request by Coeur**
Fix the comment in https://github.com/transmission/transmission/issues/6554#issuecomment-1924416885

Testing: on macOS, after running Transmission at least once, in any language (not necessarily English), you can now use Spotlight (CMD+SPACE) to search for "torrent" or "magnet" or other keywords, and it will find Transmission.
Capture d’écran 2024-02-06 à 03 28 06

### CODE REVIEW

File: macosx/en.lproj/InfoPlist.strings [hunk 1]
```diff
@@ -1,3 +1,4 @@
 /* Localized versions of Info.plist keys */
 
 NSHumanReadableCopyright = "Copyright © 2005-2024 The Transmission Project";
+MDItemKeywords = "tor,torrent,torrents,magnet,magnets,bt,bit,btih";
```
Comment (nevack) [inline macosx/en.lproj/InfoPlist.strings:4]: 1. `tor` seems unrelated, see https://www.torproject.org/
2. Do we need plural forms for `torrent` and `magnets`?
3. What are `bt`, `bit`, `btih` for? Are they for `magnet:?` query params? Isn't it already covered by `magnet` keyword itself?

Comment (Coeur) [reply macosx/en.lproj/InfoPlist.strings:4]: It doesn't matter if tor is unrelated: users of spotlight only want to write 2 or 3 characters to find an app, so here "tor" is short for torrent.
**Spotlight keywords are with exact matches only.** So you need to declare both singular and plural if you want discoverability for both.
bt and bit are short for bittorrent, the protocol
btih is the magnet protocol

Comment (nevack) [reply macosx/en.lproj/InfoPlist.strings:4]: 1. Will Spotlight prioritize "Tor Browser" over "Transmission" in this case?
2. Okay, but I just relied on [usages](https://github.com/search?q=%22MDItemKeywords%22+AND+%28+path%3A*.plist+OR+path%3A*.strings+%29+AND+NOT+is%3Afork&type=code) in other projects.

So, aside from `magnet` and `torrent`, this seems like aggressive SEO strategies :D

Comment (Coeur) [reply macosx/en.lproj/InfoPlist.strings:4]: Spotlight offers a list of results, and it prioritizes app names (so Tor Browser will show before Transmission).
Capture d’écran 2024-02-13 à 21 23 58

I'll remove btih which is too obscure.

**Review by nevack — APPROVED** [review]
I do not fully agree with the list of keywords, but overall LGTM

**Comment by Coeur** [comment]
Notes: Support finding Transmission in Spotlight with keywords "torrent" and "magnet".
